src/frontend/js/types/index.ts

```diff
@@ -7,6 +7,8 @@ export interface CourseRun {
   enrollment_end: string;
   languages: string[];
   state: CourseState;
+  starts_in_message: string;
```

Comment:
Both properties can be null.
```suggestion
  starts_in_message: Nullable<string>;
  dashboard_link: Nullable<string>;
```